generate_profiles/BuildXmlFiles.groovy

```diff
@@ -131,32 +131,34 @@ def writeRules(String rulesSetName,List<Plugin> plugins,List<String> includedBug
                         tag("owasp-a2")
                     }
                     if (pattern.attribute("type").contains("XSS") || pattern.Details.text().contains('A3-Cross-Site_Scripting')) {

```

Comment:
i think there should be also an adaption to the checks if you change the tag. looks a little bit inconsistent to me
```suggestion
                    if (pattern.attribute("type").contains("XSS") || pattern.Details.text().contains('A7-Cross-Site_Scripting')) {
```

Reply:
I deleted these parts because I found no match in code.
 - A2-Broken_Authentication_and_Session_Management
 - A4-Insecure_Direct_Object_References
 - A5-Security_Misconfiguration  
 - A8-Cross-Site_Request_Forgery
 - A7-Missing_Function_Level_Access_Control

This no longer exists in owasp 2017.
 - A9-Using_Components_with_Known_Vulnerabilities
 - A10-Unvalidated_Redirects_and_Forwards